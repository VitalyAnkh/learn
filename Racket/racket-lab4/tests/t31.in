(intersect '(x y z) '(x a y b c))
(intersect '(x y z) '(a b z))
(intersect '(green eggs) '(and ham))
(intersect '(stewed tomatoes and macaroni) '(macaroni and cheese))
