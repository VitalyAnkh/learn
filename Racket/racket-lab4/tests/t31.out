'(x y)
'(z)
'()
'(and macaroni)
